Xamarin.Forms.Platform.iOS/Renderers/SwitchRenderer.cs

```diff
@@ -78,13 +104,44 @@ void OnElementToggled(object sender, EventArgs e)
 			Control.SetState(Element.IsToggled, true);
 		}
 
+		protected override void SetBackgroundColor(Color color)
+		{
+			if (IsElementOrControlEmpty)
+				return;
+
+			if (!Forms.IsiOS13OrNewer)
+			{
+				HandleOffColorForOlderSystemVersions(Element.BackgroundColor);
+				return;
+			}
+
+			base.SetBackgroundColor(color);
+		}
+
+		void HandleOffColorForOlderSystemVersions(Color backgroundColor)
+		{
+			if (Element.OffColor == Color.Default || backgroundColor != Color.Default)
+			{
+				Control.TintColor = _defaultOffColor;
+				Control.Layer.CornerRadius = 0;
+				base.SetBackgroundColor(backgroundColor);
+				return;
```

Comment:
I don't think I quite follow here what you're saying about maintaining backwards compatibility. 

If you remove this return than the offcolor colors nicely on pre ios13 devices

Reply:
Yes, because ```HandleOffColorForOlderSystemVersions``` works only for pre iOS13 devices.
That's impossible to set "off" color and background color for pre IOS13 devices.

Just try to remove IF's body, set background color to switch, and run **WITH** OffColor set and **WITHOUT** OffColor set. You will see the difference (colored rectangle over the Switch).
I am not sure that anybody uses background color for Switch, but I decided to preserve initial behavior.

Reply:
Alright. what about this?

```C#
Control.Subviews[0].Subviews[0].Subviews[0].BackgroundColor = offColor;
```

it seems like on pre ios13 if you go down one more subview level you can color the off color?

![image](https://user-images.githubusercontent.com/5375137/84204171-ecd5a800-aa67-11ea-92a7-15fa1929ff14.png)

Reply:
> Alright. what about this?
> 
> ```cs
> Control.Subviews[0].Subviews[0].Subviews[0].BackgroundColor = offColor;
> ```
> 
> it seems like on pre ios13 if you go down one more subview level you can color the off color?
> 
> ![image](https://user-images.githubusercontent.com/5375137/84204171-ecd5a800-aa67-11ea-92a7-15fa1929ff14.png)

Interesting.
Does it work on iOS 11/10 as well?
When I was working on this PR I didn't find any solution for older versions.

Reply:
seems like it

I tested on iphone 7 with ios 10

Reply:
LoL So then those "tricky" code is my failure. Please don't hate me so much xD

Reply:
I mean... I happened upon the solution like 2 pages deep on an SO post so it was more just luck than anything.